vm/ubpf_config.h.inc

```diff
@@ -8,5 +8,6 @@
 
 #pragma once
 
+#cmakedefine UBPF_DISABLE_RETPOLINES
```

Comment:
It seems like *this* is the recommended way to do what I did about (at MARK) manually? If so, that's awesome! I learned something new!

Reply:
Not sure to be honest, but this approach works with both CMake and Visual Studio builds.

Reply:
Great!